src/main/java/org/gridsuite/filter/server/dto/expertrule/CombinatorExpertRule.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.filter.server.dto.expertrule;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.filter.server.utils.DataType;
+
+/**
+ * @author Antoine Bouhours <antoine.bouhours at rte-france.com>
+ */
+@AllArgsConstructor
+@Getter
+@SuperBuilder
+public class CombinatorExpertRule extends AbstractExpertRule {
+
+    @Override
+    @JsonProperty(access = JsonProperty.Access.READ_ONLY)
+    public DataType getDataType() {
+        return DataType.COMBINATOR;
+    }
+
+    @Override
+    public boolean evaluateRule(String identifiableValue) {
+        return false;
```

Comment:
Should implement this method as other single rule? Take the implementation found in ExpertFilterUtils
public static <I extends Injection<I>> boolean evaluateExpertFilter(AbstractExpertRule filter, Injection<I> injection) {
// As long as there are rules, we go down the tree
if (CombinatorType.AND == (filter.getCombinator())) {
return evaluateAndCombination(filter, injection);
} else if (CombinatorType.OR == filter.getCombinator()) {
return evaluateOrCombination(filter, injection);
} else {
// Evaluate individual filters
return filter.evaluateRule(getFieldValue(filter.getField(), injection));
}
}

Reply:
It's not possible with this model to implement what you suggest. Combinator rules are a special type of rules and should be handled separately. The `AbstractExpertRule.evaluateRule()` function is meant to evaluate a rule with respect to a value not to a combinator. This applies only to boolean, enum, string or number rules. See for example the signature of `evaluateRule() `compared with `public static <I extends Injection> boolean evaluateExpertFilter(AbstractExpertRule filter, Injection injection)` that you suggest moving to `CombinatorRule.evaluateRule()`. 
We could implement in `CombinatorRule.evaluateRule(String combinatorValue)` something like `getCombinator().toString().equals(combinatorValue)` but I don't think it should be used this way as it's a particular type of rule.